protocols/yfbeta/index.json

```diff
@@ -2,5 +2,5 @@
 	"cname": "yfbeta",
 	"description": "",
 	"path": "yfbeta",
-	"isEnabled": true
+	"isEnabled": false
```

Comment:
why are we disabling this?

Reply:
Their website leads to parked domain page and they seem to be inactive